src/AirshipBindings.iOS/ApiDefinition.cs

```diff
@@ -3107,7 +3108,7 @@ interface UAMessageCenterMessageViewController : IUAWKWebViewDelegate, UAMessage
     }
 
 	// @protocol UAMessageCenterMessageViewProtocol
-	[Protocol, Model]
+	[Protocol]
```

Comment:
I've been following the `[Protocol, Model]` pattern in the tool I've been working on. What's the difference here, as opposed to all the other protocols we have bindings for?

Reply:
I think it complained here because there was no base class. So I either had to add a base class or remove Model. The protocol doesn't really have a base class, so I chose to remove Model. Things seem to work.

Reply:
Interesting. I just did some similar bluntie stuff with this very class, and its lack of a base class annotation. It's one of the few protocols in AirshipKit that doesn't explicitly implement the NSObject protocol. Which is not really a base class, but that's a whole other thing I guess.